components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -23,32 +23,63 @@
 
 #include <stdlib.h>
 #include <string.h>
+#include <stdbool.h>
 
 #include "psa_manifest/sid.h"
 #include "psa/client.h"
 #include "crypto.h"
 #include "crypto_platform_spe.h"
-
 #include "mbed_assert.h"
 
 #define MINOR_VER 1
 #define CLIENT_PSA_KEY_ID_SIZE_IN_BYTES 4
 
 MBED_STATIC_ASSERT(sizeof(psa_key_id_t) == CLIENT_PSA_KEY_ID_SIZE_IN_BYTES, "Unexpected psa_key_id_t size");
 
-psa_status_t psa_crypto_init(void)
+static psa_status_t ipc_connect(uint32_t sid, psa_handle_t *handle)
+{
+    *handle = psa_connect(sid, MINOR_VER);
+    if (*handle <= PSA_NULL_HANDLE) {
+        return (PSA_ERROR_COMMUNICATION_FAILURE);
+    }
+    return (PSA_SUCCESS);
+}
+
+static inline void ipc_close(psa_handle_t *handle)
+{
+    psa_close(*handle);
```

Comment:
adding `*handle = NULL_HANDLE` would be nice

Reply:
done